.github/workflows/test-build-flow.yml

```diff
@@ -25,7 +26,7 @@ jobs:
         uses: lucacome/docker-image-update-checker@v1.1.0
         with:
           base-image: "thecodingmachine/php:${{ matrix.php }}-v4-slim-cli"
-          image: "kellerkinder/pipeline-image:${{ matrix.php }}"
+          image: "kellerkinder/pipeline-image:PHP_${{ matrix.php }}-NODE_${{ matrix.node }}"
```

Comment:
Können wir für die Kompatibilität zu bestehenden Nutzern noch Images nur mit PHP-Version und bspw. fixiert Node 14 bauen?

Reply:
Good point, a legacy workflow is added 🙂